app/models/repository.rb

```diff
@@ -1,3 +1,7 @@
 class Repository < KalibroClient::Entities::Processor::Repository
   include KalibroRecord
+
+  def self.latest(count=1)
+    all.sort { |one, another| another.id <=> one.id }.first(count)
```

Comment:
That's what I get for writing when I'm sleepy. I misread it completely, sorry.

But I remembered just now that we can use RepositoryAttributes to look them up while actually using ActiveRecord.